apps/sasquatch/src/androidTest/java/com/microsoft/azure/mobile/sasquatch/activities/SettingsActivityTest.java

```diff
@@ -0,0 +1,203 @@
+package com.microsoft.azure.mobile.sasquatch.activities;
```

Comment:
I don't understand the difference with checkboxes. In the [onSharedPreferenceChanged](https://github.com/Microsoft/mobile-center-sdk-android/pull/401/files#diff-89c150d306386e14a229932a0c1c1758R256), I update all preferences and check status of the checkboxes in the test.